api/views/phones.py

```diff
@@ -513,14 +521,27 @@ def _handle_sms_reply(relay_number, real_phone, inbound_body):
         body=inbound_body,
         to=last_text_sender.inbound_number,
     )
+    relay_number.remaining_texts -= 1
```

Comment:
Can you update a reply SMS test to check the value of `relay_number.remaining_texts`?